AppCenter/AppCenter/Internals/Util/MSEncrypter.m

```diff
@@ -100,17 +102,29 @@ - (NSString *_Nullable)decryptString:(NSString *)string {
   }
   NSString *result = nil;
   NSData *dataToDecrypt = [[NSData alloc] initWithBase64EncodedString:string options:0];
```

Comment:
Have you tried passing a nil string to `initWithBase64EncodedString:` I just hope it doesn't throw. The doc is not clear.

Reply:
Yes. It's actually marked `nonnull` and will throw. That's why `MSEncrypter`'s interface marks APIs as `nonnull`, too. Hence the check in `MSDBLogStorage` at https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1334/files/a6dcf53a682bd747c7f16a46b0817aafd85473ef#diff-8faf03edd2017a6e6cc1e7898cf0ea74R280 for an empty or empty string.